### ISSUE TITLE
The credentials saving pop-up window is closing very quickly

### ISSUE THREAD

**Issue by Zoid-dr** [keepassxreboot/keepassxc-browser#1949]
## Expected Behavior

A pop-up window should appear to save the username and password to the database

## Current Behavior

When you first log in to [google.com](https://google.com) or [yandex.ru](https://yandex.ru), after entering your login and password, a two-factor authentication window appears. At the same time, keepass-browser does not offer to save credentials. When you re-log in, when two-factor authentication is not requested, a pop-up appears, but very quickly disappears when the site redirection takes place.

## Possible Solution

Don't close the popup window automatically after redirect

## Steps to Reproduce (for bugs)

1. Go to https://accounts.google.com/ 
2. Login to the site

## Debug info

KeePassXC - 2.7.5
KeePassXC-Browser - 1.8.7
Operating system: Linux
Browser: Chromium

**Comment by varjolintu** [comment]
Increase the number of redirects count.

**Comment by Zoid-dr** [comment]
Increasing the number of allowed redirects helped with [google.com](https://google.com). But the problem persists on another sites, for example [yandex.com](https://yandex.com). Even when the number of allowed redirects is set to infinity.

## Steps to Reproduce (for bugs)
1. Go to https://passport.yandex.com
2. Login to the site

**Comment by Zoid-dr** [comment]
I did some more testing with google.com on several computers, just to be sure. I tried several allowed redirects, values mostly infinity. And it works very unstable. Most of the time, the pop-up window is hidden after redirect. On another computer, the pop-up windows does not appear at all.

Tested on:
KeePassXC - 2.7.5
KeePassXC-Browser - 1.8.7
Operating system: Linux, FreeBSD
Browser: Chromium

**Comment by Zoid-dr** [comment]
I debugged a bit and found out that the popup is hidden after a redirect if in [`const initContentScript = async function()`](https://github.com/keepassxreboot/keepassxc-browser/blob/5c6a8d059515dd5beb9162b93a04625ace210abb/keepassxc-browser/content/keepassxc-browser.js#L805C24-L805C24) the `creds.username` variable does not contain a value. I commented out this check, and then the popup is no longer hidden. I don't know why, but sometimes when logging in, `creds.username` still contains a value. I did all the checks on https://accounts.google.com
